README.md

```diff
@@ -36,27 +30,17 @@ To get a local copy up and running follow these simple example steps.
 
 ### Deployment
 
+👤 **Author**
 
```

Comment:
[Optional] I suggest that you replace "Author" with your name because that is what best fits the context.